compendium/DeclarativeServices/test/CMakeLists.txt

```diff
@@ -102,6 +103,14 @@ include_directories(${PROJECT_BINARY_DIR}/include
   ${CppMicroServices_BINARY_DIR}/compendium/CM/include
   )
 
+set(_logservice_additional_srcs
```

Comment:
in general I'd like to avoid having a dependency on the log service implementation code. I think you can write tests without it by using fake/mock log service objects

Reply:
removed log service impl dependency